src/ApiCompat/Microsoft.DotNet.ApiCompat.Tool/SuppressableConsoleLog.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using Microsoft.DotNet.ApiCompatibility.Logging;
+using Microsoft.DotNet.ApiSymbolExtensions.Logging;
+
+namespace Microsoft.DotNet.ApiCompat.Tool
+{
+    /// <summary>
+    /// Class that can log Suppressions to the Console, by implementing ConsoleLog and ISuppressableLog.
+    /// </summary>
+    internal class SuppressableConsoleLog : ConsoleLog, ISuppressableLog
+    {
+        private readonly ISuppressionEngine _suppressionEngine;
+        public bool SuppressionWasLogged { get; private set; }
```

Comment:
`/// <inheritdoc />` is missing here

Also please add an empty line to separate the private field from the property (code convention).